docs/pipeline-api.md

```diff
@@ -5782,6 +5782,20 @@ k8s.io/apimachinery/pkg/selection.Operator
 It must be non-empty</p>
 </td>
 </tr>
+<tr>
+<td>
+<code>cel</code><br/>
+<em>
+string
+</em>
+</td>
+<td>
+<em>(Optional)</em>
+<p>CEL is a string of Common Language Expression, which can be used to conditionally execute
+the task based on the result of the expression evaluation
+More info about CEL syntax: <a href="https://github.com/google/cel-spec/blob/master/doc/langdef.md">https://github.com/google/cel-spec/blob/master/doc/langdef.md</a></p>
+</td>
+</tr>
```

Comment:
Is there a way we can say in the docs that this is still WIP. And we remove it once the feature is fully available?

Reply:
Yes, I think I have added it in the docs as @jerop suggested

Reply:
Oh did you mean update the `CEL` docstrings?

Reply:
I'm ok with adding it!

Reply:
I meant in the user docs. This is so that they know when we make a release that the feature is not ready and they should not use it until it is.

Reply:
https://github.com/tektoncd/pipeline/pull/7247/files#diff-778db8afe943ee8b2b7f3acf5ec1eb68d7bf0c926a1ddb46866125d1a4c0ce7f -- it's noted here

Reply:
Yes, @jerop has suggested we add it in the doc, and it is there.

Reply:
Thanks!